<launch>

    <node pkg="mocap_optitrack" type="mocap_node" name="mocap_node" respawn="false" launch-prefix="" required="true" output="log">
        <rosparam file="$(find mocap_optitrack)/config/mocap.yaml" command="load" />
    </node>

    <node pkg="rnw_ros" type="mocap_calib_node" name="mocap_calib_node" output="screen">
        <remap from="~uav" to="/uav/pose" />
        <remap from="~cone" to="/cone/pose" />
    </node>

    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure"/>

    <node name="rviz_traj" pkg="rviz" type="rviz" output="log" args="-d $(find rnw_ros)/config/traj.rviz" />

</launch>
